client/js/table-view.js

```diff
@@ -74,10 +76,50 @@ class TableView{
         this.formulaBarEl.addEventListener('keyup', this.handleFormulaBarChange.bind(this));
     }
 
+    renderSum(collie, total) {
+        const whichColumn = document.querySelector(`#sum-${collie}`);
+        whichColumn.textContent = total;
+    }
+    
+    sumOf(arr) {
+        return arr.reduce(((total, x) => total+=x),0);
+    }
+
+    fishForNumbers(arr) {
```

Comment:
lol you and your names.  Please rename this to something less alexander-poetic.  

It actually is legit hard to understand what you're saying when you use the flowery names.